Packs/Traceable/Integrations/Traceable/Traceable.py

```diff
@@ -65,7 +66,6 @@
 
 get_spans_for_trace_id = """{
   spans(
-    limit: $limit
```

Comment:
I reviewed it internally with my platform team and for this particular query, since we are specifically providing the ids for which we want to retrieve spans we do not need the limit set:
`filter_by_clause = Helper.construct_filterby_expression(
            trace_id_clause, span_id_clause, is_anomalous_clause
        )
        query = Template(get_spans_for_trace_id).substitute(
            starttime=Helper.start_datetime_to_string(starttime),
            endtime=Helper.end_datetime_to_string(endtime),
            limit=self.limit,
            filter_by_clause=filter_by_clause,
        )`